src/components/Card.js

```diff
@@ -12,34 +17,50 @@ export default class Card {
       .content
       .querySelector('.card')
       .cloneNode(true);
-
     return cardElement;
   }
 
-  getView() {
-    this._element = this._getTemplate();
-    this._cardImage = this._element.querySelector('.card__image');
-    this._cardTitle = this._element.querySelector('.card__title');
-    this._likeButton = this._element.querySelector('.card__like-button');
-    this._deleteButton = this._element.querySelector('.card__trash-button');
-
-    this._cardTitle.textContent = this._name;
-    this._cardImage.src = this._link;
-    this._cardImage.alt = this._name;
+  _renderLikes() {
+    if (this._likeCountElement) {
+      this._likeCountElement.textContent = this._likes.length;
+    }
+  }
 
-    // Event Listeners
+  _setEventListeners() {
     this._cardImage.addEventListener('click', () => {
       this._handleCardClick(this._name, this._link);
     });
 
     this._likeButton.addEventListener('click', () => {
       this._likeButton.classList.toggle('card__like-button_active');
```

Comment:
You need to **toggle likes** only in a `then` block after a successful response from the server because if there is an error, the user will not know about that and will see the toggled heart color.